### ISSUE TITLE
No option to disable continuous availability

### ISSUE THREAD

**Issue by dcuomo** [PowerShell/xSmbShare#17]
By default, clustered file server shares are created with continuous availability turned on.  This is not ideal for information worker workloads.

Adding a parameter to disable continuous availability would be helpful.

**Comment by johlju** [comment]
We should support the `ContinuouslyAvailable` parameter on the *-SmbShare cmdlets.

**Comment by johlju** [comment]
Parameter `ContinuouslyAvailable` will be added to the `SmbShare` resource in the module ComputerManagementDsc. This module will be deprecated once the new resource is merged into ComputerManagementDsc.

Closing this issue at this time. If someone want to resolve this in this module before it is deprecated, then please comment on this issue and I will reopen the issue.
